qpx/converters/quantms/feature_adapter.py

```
@@ -93,36 +94,454 @@ def convert(
+                        "pg_global_qvalue": pg_qval,
+                        "pg_positions": None,
+                        "ion_mobility_start": None,
+                        "ion_mobility_stop": None,
+                        "gg_accessions": gg,
+                        "gg_names": gg,
+                        "id_run_file_name": id_run,
+                        "rt_start": None,
+                        "rt_stop": None,
+                    }
+                )
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)